Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -1862,7 +1865,7 @@ class ExprWalker : PythonWalker {
                 }
 
                 yield return OutliningTaggerProvider.OutliningTagger.GetTagSpan(
-                    currentSnapshot,
+                    translator.TextBuffer.CurrentSnapshot,
```

Comment:
I don't really like the idea of this potentially changing while enumerating... why do we want to make this change?